app/Http/Controllers/PostController.php

```diff
@@ -21,42 +21,70 @@ class PostController extends Controller
      * Author: Lampham
      */
     protected $postRepository;
+    protected $answerRepository;
+    protected $tagRepository;
+    protected $userRepository;
+    protected $report;
 
-    public function __construct(PostRepositoryInterface $postRepository)
+    public function __construct(PostRepositoryInterface $postRepository,AnswerRepositoryInterface $answerRepository,
+        TagRepositoryInterface $tagRepository, UserRepositoryInterface $userRepository, Report $report)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.CloseBracketLine</br>**Severity**: error</br>**File**: app/Http/Controllers/PostController.php</br>**Line**: 30</br>The closing parenthesis of a multi-line function declaration must be on a new line